.gitignore

```diff
@@ -8,4 +8,4 @@ api-docs
 packages/*/*.tgz
 packages/*/dist*
 packages/*/package
-packages/cli/test/sandbox
+packages/*/test/sandbox
```

Comment:
I think it was a mistake to put the sandbox directory under `test`, considering that we are running mocha in recursive mode. I think we should move this sandbox to something like `packages/*/sandbox` or even `packages/*/.sandbox`. It's out of scope of your pull request though.

Reply:
I've changed it to `.sandbox` for `boot`. I'll leave the cli change to `.sandbox` for later (post-MVP).